src/components/PostInfo/PostInfo.scss

```diff
@@ -1 +1,17 @@
 // add styles here
+.PostInfo {
+    margin: 10px auto;
+    width: 90%;
+    border: 1px solid #000;
+    border-radius: 8px;
+    background-color: lightgray;
+    padding: 1em;
+
+    &__title {
+        margin: 0;
+    }
+
+    &__header {
+        margin-bottom: 1em;
+    }
+}
```

Comment:
![2024-01-23_12h52_03](https://github.com/mate-academy/react_static-list-of-posts-js/assets/61904995/8b1f49a7-5830-4cf3-8f8a-cda337207c6c)

Reply:
could you be more clear i don`t get it